cypress/integration/golden-path.common.ts

```diff
@@ -2,17 +2,15 @@ import imageSampleCollection from "../../typescript/web/src/utils/image-sample-c
 
 export const declareGoldenPathTests = ({
```

Comment:
We don't need to use an object anymore here since we've only got a single param 😉

Reply:
I left it as an object on purpose so that all functions in the graphql definitions have a similar API

Reply:
OK I see, cool 👍 

That's what I did too in #667 too with the Jest tests 😉 :
* [workspace-name-message.test.tsx](https://github.com/labelflow/labelflow/blob/feature/delete-workspace/typescript/web/src/components/workspace-name-input/workspace-name-message.test.tsx#L83)
* [create-workspace-modal.test.tsx](https://github.com/labelflow/labelflow/blob/feature/delete-workspace/typescript/web/src/components/workspace-switcher/workspace-creation-modal/create-workspace-modal.test.tsx#L95)

I even used the same names for `declareTests`, the `TestCase` type and the `TEST_CASES` constant so that files can be copy-pasted with more ease when adding a new unit-test 🚀  Maybe we could do the same here too?

Reply:
I did something similar in 7615c26d8f0457a529d44ec4925c1cc9a5b60507, let me know if it's what you were expecting!

Reply:
Yes that's perfect 👌 Thank you 🙂